processing-service-core/api/models/globalConfig.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var Q = require('q');
+var repository = require('../store/globalConfigStore.js');
+var logger = require('../helpers/loggingHelper.js');
+
+
+/**
+ * Creates or updates global config based on the globalConfigParams.
+ * 
+ * @param {String} globalConfigParams The json string represents key-value pair map of global configuration.
+ * @returns {*|d.promise|Function|promise|a|h} a promise that will be resolved or rejected based on result of create/update.
+ */
+module.exports.createGlobalConfig = function(globalConfigParams){
```

Comment:
Instead of having each method prefixed with "module.exports." can you just put these in an object at the top of each class. It just makes it a little clearer.